docs/live-preview.md

```diff
@@ -0,0 +1,96 @@
+---
+py-config:
+  packages:
+    - myst-docutils
+    - docutils==0.19
+    - pygments
+  splashscreen:
+    autoclose: true
+---
+
+# Live Preview
+
+This is a live preview of the MyST Markdown [docutils renderer](docutils.md).
+You can edit the text/configuration below and see the live output.
```

Comment:
```suggestion
You can edit the text/configuration below and see the live output.
Note that there is no styling applied (that is usually provided by Sphinx themes).
```

Reply:
FYI, you can specify the CSS, I changed it in the config from the built-in default `plain.css` to `minimal.css`, which is a little nicer 😄